src/kill_switch.cairo

```diff
@@ -0,0 +1,42 @@
+#[starknet::interface]
+pub trait IKillSwitch<TContractState> {
+    fn is_active(self: @TContractState) -> bool;
+    fn activate(ref self: TContractState); // this turns the is_active variable to true
+    fn deactivate(ref self: TContractState); // this turns the is_active variable to false
+}
+
+#[starknet::contract]
+mod KillSwitch {
+    use super::IKillSwitch;
+
+    #[storage]
+    struct Storage {
+        is_active: bool,
+    }
+
+    #[constructor]
```

Comment:
Great job @EjembiEmmanuel 
Glad you've understanding of how to use dispatchers. 
However, I would like you to:  
- remove the `constructor` of the `KillSwitch` contract
- When you call `increase_aggr_count_by_two`, make sure you utilize the `activate` function of `KillSwitch` to manually turn ` is_active` variable of your `KillSwitch` contract to true directly within `AggregatorContract`.